server/public/styles.css

```diff
@@ -1,5 +1,38 @@
+body {
+  max-width: 1000px;
+  margin: 0 auto;
+}
+
+body> div {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+}
 header {
   display: flex;
   justify-content: space-between;
   align-items: flex-end;
+  flex-basis: 100%;
+}
+
+table {
+  flex-basis: 60%;
+}
+form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-basis: 30%;
+}
+
+label {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
```

Comment:
Check out Bootstrap's flex utility classes. You can probably reduce your custom CSS by a lot.

https://getbootstrap.com/docs/4.4/utilities/flex/